.github/workflows/python.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package CI
+
+on: [ push, pull_request ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel ruff
+
+      - name: Run ruff
+        run: ruff check nucypher_ops
```

Comment:
Looking at `nucypher`'s use of ruff (see https://github.com/nucypher/nucypher/blob/main/.github/workflows/ruff.yaml), should this be?
```
ruff --output-format=github nucypher_ops
```

Just wondering if there is a reason for the discrepancy.

Reply:
No reason, I don't know what `--output-format=github` does.

Reply:
I believe (?) the format caters to specific output that can be viewed as gh actions workflow outputs. Eg. https://github.com/nucypher/nucypher/actions/runs/6708762664/job/18230283150